### DISCUSSION TITLE
Possible to make network share-aware?

### DISCUSSION THREAD

**Discussion by beaudamore** [hassio-addons/addon-plex#195] [Ideas]
Hello,
I was using this other mod of this plugin at https://github.com/dianlight/addon-plex where he rolled up CIFS so that Plex can actually 'see' NAS files.
It has been deprecated for some time now.
I just managed to get an automation kicked off at startup to mount a network drive to the media folder via:

`shell_command:
  mount_serenitynas_mediafolder: mkdir -p /media/nas_media;mount -t cifs -o vers=3.0,noserverino,username=xxx,password=xxx,domain=WORKGROUP //192.168.134.245/Media /media/nas_media`

Navigating to the media/nas/media then shows me the folder/files available in the NAS ok...

However, when trying to add a new library to Plex, it doesn't see anything inside the 'nas_media' folder... The dianlight forked addon made a new choice in the source list when adding a new library to Plex as /IP of the CIFS share it creates on it's own...

Is there a modification to the command above that will direct Plex to 'follow the trail' into the share as it were?

Or is there a possibility of adding in the mods that dianlight made in that other fork to the main fork?

I would love to do it myself, however, I struggle with Linux coming from a mainly Microsoft background...

Thanks for reading!

**Comment by frenck** [comment]
That is not a task for add-on, but for the system.

../Frenck

**Reply by beaudamore** [reply]
Thank you for the reply. I appreciate it.
Plex does not see the folders/files in the share I made with CIFS.
Media browser can see them though. Strange.
I guess I will start looking through the source of dianlight's fork and see if I can see what was done to remedy this.
